src/components/VideoList.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from "react";
+import {useVideo} from "../context/video-context";
+import {VideoCard} from "./VideoCard";
+import "./VideoList.css";
+import {VideoPage} from "./VideoPage";
+export function VideoList({value, title = null}) {
+  // const {value} = useVideo();
+  const [showVideoPage, setShowVideoPage] = useState(false);
+  const [videoId, setVideoId] = useState("");
+  const newTitle = title ?? "All Videos";
```

Comment:
Transform the title to uppercase here itself

```suggestion
const newTitle = title.toUpperCase() ?? "All Videos".toUpperCase();
````